mod-finance/schemas/fund_update_log.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Fund update logs",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "id of the fund update log record",
+      "type": "integer"
+    },
+    "_version": {
+      "type": "integer",
+      "description": "Record version for optimistic locking"
+    },
+    "jobName": {
+      "description": "The name of the job",
+      "type": "string"
+    },
+    "jobDetails": {
+      "description": "details of job",
+      "type": "object"
```

Comment:
for now it is okay to be an object, but in future we need to define this with appropriate set of fields